src/dipdup/dipdup.py

```diff
@@ -336,14 +336,9 @@ async def _execute_sql_scripts(self, reindex: bool) -> None:
         if not exists(sql_path):
             return
         self._logger.info('Executing SQL scripts from `%s`', sql_path)
-        for filename in sorted(listdir(sql_path)):
-            if not filename.endswith('.sql'):
-                continue
-
-            with open(join(sql_path, filename)) as file:
-                sql = file.read()
-
-            self._logger.info('Executing `%s`', filename)
+        for file in iter_files(sql_path, '.sql'):
+            self._logger.info('Executing `%s`', file.name)
+            sql = file.read()
             await get_connection(None).execute_script(sql)
```

Comment:
Typical issue: commented content thus the file is not empty but the script is invalid

Reply:
Check for "de facto empty" files would be too complex — GraphQL has multiline comments. Wrapped exception thrown on empty SQL.